rules/scala/compiler/Arguments.scala

```diff
@@ -60,7 +71,7 @@ object Arguments {
       .addArgument("--log_level")
       .help("Log level")
       .choices(LogLevel.Debug, LogLevel.Error, LogLevel.Info, LogLevel.None, LogLevel.Warn)
-      .setDefault_(LogLevel.Info)
+      .setDefault_(LogLevel.Warn)
```

Comment:
Which logging level do you recommend as the default for Zinc?

Reply:
`info` is the best default, otherwise users can miss some important logs coming from Zinc. I suggest that this is configured via a bazel CLI arg so that it can accommodate the likes of people don't want to see the "Compiling..." logs (I assume this is the primary motivation for this change).

Reply:
Info is default for sbt, I think.

I can't find the reference now, but Bazel convention is that a successful action has no output. (This works best with the CLI, plus the fact that bazel builds can involve thousands of actions.)

This would mean warn or error would be appropriate. I agree it should be configurable, though I think the default should be warn.